### DISCUSSION TITLE
How can I replicate from PlanetScale to MySQL on an external server?

### DISCUSSION THREAD

**Discussion by tomato-ga** [planetscale/discussion#577] [Q&A] [answered]
How can I replicate from PlanetScale to MySQL on an external server?

I want to store data stored on PlanetScale as a replica on my Home server.

**Comment by orware** [comment, marked as answer]
Hello @tomato-ga,

There are a few ways you can go about this, although both of the approaches below aren't real-time replication options they are the options available if you did want to try and bring data into a local home server relatively easily.

One method would be to utilize our Airbyte source connector available from here:
https://planetscale.com/docs/integrations/airbyte

And use it to send your PlanetScale database data into that external MySQL server using Airbyte's MySQL destination connector.

That would allow you to more or less keep your external MySQL server in sync with the PlanetScale database so long as you have Airbyte running a sync on a regular schedule.

A second method you could use would be to use our `pscale database dump` command (https://planetscale.com/docs/reference/database), along with the [MyDumper](https://github.com/mydumper/mydumper) project's `myloader` utility. You could script things out to generate a local back up first using `pscale database dump` and then load it into your local MySQL server using `myloader`. You could then have that script run semi-frequently to generate an updated dataset and then replace the current local MySQL copy with the newest version.

**Reply by tomato-ga** [reply]
This is what Airbyte was looking for. Thank you!
